tests/e2e/distributed_queries/distributed_queries.py

```diff
@@ -48,11 +48,11 @@ def test_vertex_creation_and_scanall(connection):
     wait_for_shard_manager_to_initialize()
     cursor = connection.cursor()
 
-    assert has_n_result_row(cursor, "CREATE (n :label {property:1, asd:2})", 0)
-    assert has_n_result_row(cursor, "CREATE (n :label {property:2, asd:2})", 0)
-    assert has_n_result_row(cursor, "CREATE (n :label {property:3, asd:2})", 0)
-    assert has_n_result_row(cursor, "CREATE (n :label {property:4, asd:2})", 0)
-    assert has_n_result_row(cursor, "CREATE (n :label {property:5, asd:2})", 0)
+    assert has_n_result_row(cursor, "CREATE (n :label {property:1})", 0)
+    assert has_n_result_row(cursor, "CREATE (n :label {property:2})", 0)
+    assert has_n_result_row(cursor, "CREATE (n :label {property:3})", 0)
+    assert has_n_result_row(cursor, "CREATE (n :label {property:4})", 0)
+    assert has_n_result_row(cursor, "CREATE (n :label {property:5})", 0)
```

Comment:
We don't initialize the `asd` property in the `ShardMap`, so we were just lucky because it didn't cause any issues (dereferencing an optional without actual value).